fvm/flex/utils/testBackend.go

```diff
@@ -0,0 +1,233 @@
+package utils
+
+import (
+	"encoding/binary"
+	"testing"
+
+	"github.com/onflow/atree"
+	"github.com/onflow/cadence"
+	"github.com/onflow/cadence/runtime/common"
+
+	fvmenv "github.com/onflow/flow-go/fvm/environment"
+	"github.com/onflow/flow-go/fvm/flex/models"
+	"github.com/onflow/flow-go/fvm/meter"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func RunWithTestBackend(t testing.TB, f func(models.Backend)) {
+	tb := testBackend{
+		testValueStore:   getSimpleValueStore(),
+		testEventEmitter: getSimpleEventEmitter(),
+	}
+	f(tb)
+}
+
+func getSimpleValueStore() *testValueStore {
+	data := make(map[string][]byte)
+	allocator := make(map[string]uint64)
+
+	return &testValueStore{
+		getValue: func(owner, key []byte) ([]byte, error) {
+			return data[string(owner)+"~"+string(key)], nil
```

Comment:
Maybe create a function for the repeated key creation expressions here and in the other functions below (concatenate owner and key with `~`)